jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricherTest.java

```diff
@@ -307,28 +316,28 @@ private Map<String, Map<String, Object>> createFakeConfig(String config) {
     }
 
     private void assertHttpGet(Probe probe, String scheme, String path, int port) {
-      assertThat(probe).isNotNull()
-          .extracting(Probe::getHttpGet)
-          .hasFieldOrPropertyWithValue("host", null)
-          .hasFieldOrPropertyWithValue("scheme", scheme)
-          .hasFieldOrPropertyWithValue("path", path)
-          .hasFieldOrPropertyWithValue("port.intVal", port);
+        assertThat(probe).isNotNull()
+                .extracting(Probe::getHttpGet)
+                .hasFieldOrPropertyWithValue("host", null)
+                .hasFieldOrPropertyWithValue("scheme", scheme)
+                .hasFieldOrPropertyWithValue("path", path)
+                .hasFieldOrPropertyWithValue("port.intVal", port);
     }
 
     private void assertProbeAdded(Probe probe, String path, int port,
                                   int initialDelay, int failureThreshold, int successThreshold, int periodSeconds) {
-      assertHttpGet(probe, "HTTPS", path, port);
-      assertThat(probe).isNotNull()
-          .returns(initialDelay, Probe::getInitialDelaySeconds)
-          .returns(failureThreshold, Probe::getFailureThreshold)
-          .returns(successThreshold, Probe::getSuccessThreshold)
-          .returns(periodSeconds, Probe::getPeriodSeconds);
+        assertHttpGet(probe, "HTTPS", path, port);
+        assertThat(probe).isNotNull()
+                .returns(initialDelay, Probe::getInitialDelaySeconds)
+                .returns(failureThreshold, Probe::getFailureThreshold)
+                .returns(successThreshold, Probe::getSuccessThreshold)
+                .returns(periodSeconds, Probe::getPeriodSeconds);
```

Comment:
Could you please revert these formatting changes?

Reply:
- Yeah I was formatting the line for some lines but if git effected I'll revert these changes